composer.json

```
@@ -28,7 +28,9 @@
         "roave/no-floaters": "^1.1",
         "phpstan/phpstan": "^0.11.5",
         "symfony/var-dumper": "^4.2",
-        "thecodingmachine/phpstan-strict-rules": "^0.11.0"
+        "thecodingmachine/phpstan-strict-rules": "^0.11.0",
+        "phpunit/phpunit": "^7.5|^8.0",
```

I would bump this phpunit 8 only.

```suggestion
        "phpunit/phpunit": "^8.0",
```